src/toxcore/dht/server/mod.rs

```diff
@@ -220,9 +220,16 @@ impl Server {
     }
 
     /// Add a friend.
-    pub fn add_friend(&self, friend: DhtFriend) {
+    /// `node_to_bootstrap` of new friend is filled with close nodes for fast bootstrapping.
+    pub fn add_friend(&self, mut friend: DhtFriend) {
```

Comment:
I suggest to create `DhtFriend` inside this method and take `PublicKey` only. We don't need to do anything with a `DhtFriend` outside and it can be hidden.

Reply:
done